pandasdmx/utils/__init__.py

```diff
@@ -89,7 +89,13 @@ def __call__(self, name, fields):
         return namedtuple class as singleton 
         """
 
-        fields = tuple(fields)
+        _fields = []
+        for field in fields:
+            if "-" in field:
+                field = field.replace("-", "_")
+            _fields.append(field)

```

Comment:
If you will wait before confirming or rejecting the PR. I will revive tests to find the error and see if there is not another way to resolve it. Thank you.